views/home.html.erb

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en" class="mdl-js">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="assets/favicon.ico" rel="icon">
+    <link href="assets/css/bootstrap.min.css" rel="stylesheet">
+    <link href="assets/css/styles.css" rel="stylesheet">
+    <script src="assets/js/jquery.min.js"></script>
+    <script src="assets/js/bootstrap.min.js"></script>
+    <title><%= I18n.t(:app_name) %></title>
+  </head>
+  <body>
+    <section id="introduction">
+      <div class="container">
+        <div class="row fill-viewport align-items-center justify-content-center">
+          <div class="col-12 col-md-6">
+            <div class="row">
+              <div class="col-md-auto">
+                <h1 class="text-white title">Codebreaker 2018</h1>
+              </div>
+            </div>
+            <div class="row">
+              <p class="lead text-white-70">
+                Try to guess 4-digit number, that consists of numbers in a range from 1 to 6.
```

Comment:
If u r using I18n, u need to move all views text to I18n

Reply:
done